automatic/dropbox/tools/chocolateyuninstall.ps1

```diff
@@ -1,10 +1,23 @@
-. (Join-Path (Split-Path $MyInvocation.MyCommand.Definition -Parent) 'helpers.ps1')
+$ErrorActionPreference = 'Stop'
```

Comment:
If you removed BOM here, please add it back.
Powershell incorrectly reads script files as ANSI when no BOM is available.